src/Apps/Search/Routes/Artworks/Components/Filter/Filters/ColorFilter.tsx

```diff
@@ -58,134 +54,131 @@ const CheckmarkPositions = {
   },
 }
 
-export class ColorFilter extends React.Component<Props> {
-  toggleColor(color) {
-    const { filters } = this.props
-    if (filters.state.color === color) {
-      filters.unsetFilter("color")
+export const ColorFilter: FC = () => {
```

Comment:
All of these individual filter components were rewritten as FC's, using `useFilterContext` instead of having the filters passed in as props. This probably isn't necessary for the refactor, but it made me feel good.

Reply:
Lovely 👌